components/pd_client/src/metrics.rs

```diff
@@ -48,6 +48,20 @@ make_static_metric! {
         try_connect,
     }
 
+    pub label_enum StoreSizeEventType {
+        capacity,
+        available,
+        used,
+        snap_size,
+        raft_size,
+        kv_size,
```

Comment:
IMO, `kv_size` is a little bit redundant, which always accounts the major size. 

And it can be calculated by `used - snap_size - raft_size - import_size`.

Reply:
It's not accurate if the user put the file into the directory and we resever on  placehold file(100GB) in the directory